src/app.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import Radium from 'radium'
+import invariant from 'invariant'
+
+import {
+  Subject
+} from 'rx'
+
+import {
+  connection,
+  status
+} from 'pipboylib'
+
+const {
+  discover,
+  createSocket,
+  sendPeriodicHeartbeat,
+  createConnectionSubject
+} = connection
+
+const {
+  connected
+} = status
+
+import {
+  toType
+} from './constants/server_types'
+
+import dispatcher from './dispatcher'
+
+const styles = {
+  app: {
+    display: 'block',
+    height: '100%',
+    width: '100%'
+  }
+}
+
+@Radium
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  static childContextTypes = {
+    sendCommand: React.PropTypes.func.isRequired
+  }
+
+  state = {
+    connected: false
+  }
+
+  getChildContext = () => ({
+    sendCommand: this.sendCommand
+  })
+
+  sendCommand = (type, ...args) => {
+    if (this.connection) {
+      this.connection.onNext([type, ...args])
+    }
+  }
+
+  componentWillMount() {
+    // TODO: Implement a server selection
+    discover()
```

Comment:
Well I'm planning to write some command actions that will be injected into the subject, thus complying to the Flux scheme